docs/Search-auxiliary-files.md

```diff
@@ -0,0 +1,170 @@
+# Search auxiliary files
+
+**Subsystem: Search 🔎**
+
+Aside from metadata stored in the [Azure Search indexes](Azure-Search-indexes.md), there is data stored in Azure Blob
+Storage for bookkeeping and performance reasons. These data files are called **auxiliary files**. The data files
+mentioned here are those are explicitly managed by the search subsystem. Other data files exist (manually created,
+created by the statistics subsystem, etc.). Those will not be covered here but are mentioned in the job-specific
+documentation that uses them as input.
+
+Each search auxiliary file is copied to the individual region that a [search service](../src/NuGet.Services.SearchService/README.md)
+is deployed. For nuget.org, we run search in four regions, so there are four copies of each of these files.
+
+The search auxiliary files are:
+
+  - [`downloads/downloads.v2.json`](#download-count-data) - total download count for every package version
+  - [`owners/owners.v2.json` and change history](#package-ownership-data) - owners for every package ID
+  - [`verified-packages/verified-packages.v1.json`](#verified-packages-data) - package IDs that are verified
+  - [`popularity-transfers/popularity-transfers.v1.json`](#popularity-transfer-data) - popularity transfers between package IDs
+
+## Download count data
+
+The `downloads/downloads.v2.json` file has the total download count for all package versions. The total download count
+for a package ID as a whole can be calculated simply by adding all version download counts.
+
+The downloads data file looks like this:
+
+```json
+{
+  "Newtonsoft.Json": {
+    "8.0.3": 10508321,
+    "9.0.1": 55801938
+  },
+  "NuGet.Versioning": {
+    "5.6.0-preview.3.6558": 988,
+    "5.6.0": 10224
+  }
+}
+```
+
+The package ID and version keys are not guaranteed to have the original (author-intended) casing and should be treated
+in a case insensitive manner. The version keys will always be normalized via standard `NuGetVersion` normalization rules
```

Comment:
Would a link to `NuGetVersion` normalization rules be justified here?
https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#normalized-version-numbers

Reply:
Done